src/dsymbol/conversion/package.d

```diff
@@ -143,17 +143,38 @@ class SimpleParser : Parser
 		return null;
 	}
 
-	override FunctionBody parseFunctionBody()
+	override MissingFunctionBody parseMissingFunctionBody()
 	{
-		bool needDo;
-		// no impl
+		skipContracts();
+		if (moreTokens && (currentIs(tok!"do") || current.text == "body"))
```

Comment:
You must also return null if cirrent is open brace (body begings). But when i test this then i got crashes in the ast formatter which seems to contain unsafe accesses.